cacti/README.md

```diff
@@ -9,107 +9,142 @@ Get metrics from cacti service in real time to:
 
 ## Setup
 
-Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][1] for guidance on applying these instructions.
-
 ### Installation
 
-The Cacti check is included in the [Datadog Agent][2] package, to start gathering metrics you first need to:
-- Install librrd headers and libraries
-- Install python bindings to rrdtool
+The Cacti check is included in the [Datadog Agent][1] package, to start gathering metrics you first need to:
+
+1. Install `librrd` headers and libraries.
+2. Install python bindings to `rrdtool`.
 
 #### librrd headers and librairies
 
-On Debian/Ubuntu
+On Debian/Ubuntu:
+
 ```shell
 sudo apt-get install librrd-dev
 ```
 
-On RHEL/CentOS
+On RHEL/CentOS:
+
 ```shell
 sudo yum install rrdtool-devel
 ```
 
 #### Python bindinges
```

Comment:
Not related to this PR, but maybe this typo is within the scope of formatting updates? :-)

```suggestion
#### Python bindings
```